sdk/test-utils/recorder/src/recorder.ts

```diff
@@ -216,6 +217,12 @@ export class Recorder {
    * - sanitizerOptions - Generated recordings are updated by the "proxy-tool" based on the sanitizer options provided, these santizers are applied only in "record" mode.
    */
   async start(options: RecorderStartOptions): Promise<void> {
+    if (isBrowser && isPlaybackMode()) {
```

Comment:
Move this to a new method "preStart()" or similar to keep it cleaner?

Reply:
Updated